### PR TITLE
Update CHANGELOG with the changes for 1.1.2.

### PR DESCRIPTION
**Pull request by sersut**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
+* New notification type "Splunk"
+  * New Splunk configuration page in the Analytics UI.
+  * Ability to handle "Splunk" aliases.
+  * This feature is disabled by default. Can be enabled by:
+    * Add `features['integration'] = true` to `/etc/opscode-analytics/opscode-analytics.rb`.
+    * Run `opscode-analytics-ctl reconfigure`.
+*
+
+Users can configure Chef Analytics to send data out to Splunk Servers.
+
```
Comment (jamesc) [inline CHANGELOG.md]: Should we add in the SSL omnibus-software change too ?

**Comment by sersut** [comment]
@jamesc done!

@abuecker do you want to include Node State Dashboard in the CHANGELOG?

**Comment by metadave** [comment]
1.1.1 is out, should this PR be merged?

**Comment by sersut** [comment]
I was holding off on this until Wednesday but I think yes we can do it. In fact:

**Comment by sersut** [comment]
:smile:

**Pull request merged**